### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by mit-mit**
Initial travis config, based on https://github.com/dart-lang/source_gen/blob/master/.travis.yml

Fixes https://github.com/dart-lang/grpc-dart/issues/3

### CODE REVIEW

**Comment by mit-mit** [comment]
Test build queued: https://travis-ci.org/dart-lang/grpc-dart/builds/248370131

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Run against dev and stable channels.
+dart:
+  - dev
+  - stable
+# Define test tasks to run.
+dart_task:
+  # Run the tests -- include the default-skipped presubmit tests.
+  - test: --platform vm --run-skipped
+  - dartfmt
+  - dartanalyzer
```
Comment (jakobr-google) [inline .travis.yml]: In https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml, the dartfmt and dartanalyzer tasks are only run once per platform (dev/stable). Do we want to do the same here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# Only building master means that we don't run two builds for each pull request.
+branches:
+  only: [master]
+
```
Comment (jakobr-google) [inline .travis.yml]: We might also want to add

```
os:
  - linux
  - osx
```

**Comment by mit-mit** [comment]
New build queued: https://travis-ci.org/dart-lang/grpc-dart/builds/248414629

**Comment by mit-mit** [comment]
Seems to be working, merging

**Comment by mit-mit** [comment]
@jakobr-google note that it found some analysis issues

**Pull request merged**